### PR TITLE
Medical Module Additions

### PR DESCRIPTION
**Pull request by Saforax**
Changed AGM Medical Module to support the choice of medical item which requires a MedEvac. So now the user can choose what they want to have a MedEvac for between Bloodbag/Epipen. The decision came about from my community wanting blood bags instead.

### CODE REVIEW

File: AGM_Medical/functions/fn_module.sqf [hunk 1]
```diff
@@ -41,6 +41,7 @@ publicVariable "AGM_Medical_MEDEVACVehicles";
 [_logic, "AGM_Medical_AllowChatWhileUnconscious",    "AllowChatWhileUnconscious"   ] call AGM_Core_fnc_readBooleanParameterFromModule;
 [_logic, "AGM_Medical_EnableOverdosing",             "EnableOverdosing"            ] call AGM_Core_fnc_readBooleanParameterFromModule;
 [_logic, "AGM_Medical_RequireMEDEVAC",               "RequireMEDEVAC"              ] call AGM_Core_fnc_readBooleanParameterFromModule;
+[_logic, "AGM_Medical_TypeMEDEVAC",               "TypeMEDEVAC"              ] call AGM_Core_fnc_readNumericParameterFromModule;
```
Comment (Saforax) [inline AGM_Medical/functions/fn_module.sqf]: Could be moved in order to be with other Numeric types

**Comment by KoffeinFlummi** [comment]
Thanks for your contribution!

Two things:
1. Please add yourself to the AUTHORS file.
2. See below.

File: AGM_Medical/functions/fn_treat.sqf [hunk 2]
```diff
@@ -33,6 +34,13 @@ if (_type in ["epipen", "bloodbag"] and
   };
 };
 
+switch(AGM_Medical_TypeMEDEVAC) do {
+  case 0 : {_typeMedevac = "epipen"};
+  case 1 : {_typeMedevac = "bloodbag"};
+  default {_typeMedevac = "epipen"};
+};
```
Comment (KoffeinFlummi) [inline AGM_Medical/functions/fn_treat.sqf]: This could be replaced with

```sqf
_typeMedevac = ["epipen", "bloodbag"] select AGM_Medical_TypeMEDEVAC
```

**Comment by Saforax** [comment]
Done those changes as suggested.

**Comment by KoffeinFlummi** [comment]
Thanks!

**Pull request merged**